clang/test/Driver/sycl-offload-with-split.c

```diff
@@ -204,7 +196,7 @@
 // RUN:  | FileCheck %s -check-prefixes=CHK-TOOLS-AOT,CHK-TOOLS-GEN
 // RUN: %clang -target x86_64-unknown-linux-gnu -fsycl -fno-sycl-device-lib=all -fsycl-device-code-split -fsycl-targets=spir64_x86_64-unknown-unknown-sycldevice %s -### 2>&1 \
 // RUN:  | FileCheck %s -check-prefixes=CHK-TOOLS-AOT,CHK-TOOLS-CPU
-// CHK-TOOLS-AOT: clang{{.*}} "-fsycl-is-device" {{.*}} "-o" "[[OUTPUT1:.+\.bc]]"
+// CHK-TOOLS-AOT: clang{{.*}} "-fsycl-is-device"{{.*}} "-fsycl-int-header=[[INPUT1:.+\.h]]"{{.*}} "-o" "[[OUTPUT1:.+\.bc]]"
```

Comment:
I'm not 100% sure if the `-header` postfix is anything more significant than an implementation detail - leaving a bunch of related suggestions just in case.
```suggestion
// CHK-TOOLS-AOT: clang{{.*}} "-fsycl-is-device"{{.*}} "-fsycl-int-header=[[INPUT1:.+\-header\.h]]"{{.*}} "-o" "[[OUTPUT1:.+\.bc]]"
```